upload_build.sh

```diff
@@ -51,7 +54,7 @@ if [[ $ARCH == "windows" ]]; then
   FILE=$BASE.zip
   ls /mingw64/bin/
   # This list was produced by `ldd livepeer.exe`
-  LIBS="libffi-7.dll libgcc_s_seh-1.dll libgmp-10.dll libgnutls-30.dll libhogweed-6.dll libiconv-2.dll libidn2-0.dll libintl-8.dll libnettle-8.dll libp11-kit-0.dll libtasn1-6.dll libunistring-2.dll libwinpthread-1.dll zlib1.dll"
+  LIBS="libffi-6.dll libgcc_s_seh-1.dll libgmp-10.dll libgnutls-30.dll libhogweed-4.dll libiconv-2.dll libidn2-0.dll libintl-8.dll libnettle-6.dll libp11-kit-0.dll libtasn1-6.dll libunistring-2.dll libwinpthread-1.dll zlib1.dll"
```

Comment:
Curious where this library downgrade is coming from? Also, I know this was around in the old build, but has it been an issue to have dll dependencies - does this mean the user also needs a particular version of gnutls etc?

Reply:
The version of MSYS2 that's installed from the GitHub actions is slightly older than what we were using before, so a few packages are a bit older. The advantage there is that they should fluctuate less, resulting in fewer CI failure package bump chores like https://github.com/livepeer/go-livepeer/pull/1488

It hasn't been an issue 'cause we package all the DLL dependencies in the zip file with the binaries, so there are no external dependencies on the end user as long as they execute livepeer.exe from the folder containing all the DLLs. I'd love to move to a fully-static Windows build someday, but I think that'd mean [building everything on this list from source using PKGBUILD](https://www.msys2.org/wiki/Creating-Packages/) which is not so fun.